src/components/datepicker/Datepicker.scss

```diff
@@ -0,0 +1,176 @@
+@import '~hds-core/lib/components/text-input/text-input.css';
+
+.datepickerWrapper {
```

Comment:
I left the naming of these datepicker classnames intact with the original. Should we change it to our style (kebab-style)? I think we should apply the HDS datepicker once it is ready, so I kind of thought it might not be worth the effort. Then again, it might take a long time for HDS datepicker to be ready, and having 2 different styles of classnames in the codebase is not optimal, and might be confusing 🤔

Reply:
This is fine since it is extracted to its own module 👍